compiler/x/codegen/OpBinary.cpp

```diff
@@ -911,3 +911,904 @@ const TR_X86OpCode::TR_OpCodeBinaryEntry TR_X86OpCode::_binaryEncodings[IA32NumO
       {0x00, 0x00, 0x00, 0},    // AdjustFramePtr
       {0x00, 0x00, 0x00, 0}     // ReturnMarker
    };
+
+#include "codegen/FrontEnd.hpp"
+bool TR_X86OpCode::_UseDeprecatedBinaryEncoding = feGetEnv("TR_UseDeprecatedBinaryEncoding");
+// X87 is only available on X86-32
+#ifdef TR_TARGET_64BIT
+#define X87_________________(x) 0,0,0
+#else
+#define X87_________________(x) (uint8_t)((x & 0xE0) >> 5), (uint8_t)((x & 0x18) >> 3), (uint8_t)(x & 0x07)
+#endif
+
+const TR_X86OpCode::OpCode_t TR_X86OpCode::_binaries[] =
+{
+    { VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_____, 0xcc, 0, ModRM_NONE, Immediate_0 },    // BADIA32Op/int 3
```

Comment:
There should be a comment above this table listing the fields and a reference to where these contstants are defined so that they can be easily referenced - full understanding of the entries should be possible without hunting for the prefixes.